Src/BootCamp.Chapter/FileCleaner.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class FileCleaner
+    {
+        private string _dirtyFile;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_dirtyFile' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)

Reply:
changed